hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/tests/hdfspp_mini_dfs_smoke.cc

```diff
@@ -34,7 +34,6 @@ TEST_F(HdfsMiniDfsSmokeTest, SmokeTest) {
   EXPECT_NE(nullptr, connection.handle());
 }
 
-
```

Comment:
Avoid

Reply:
I need to touch at least one test file. Otherwise, `test4tests` will fail while building for the first platform in the pipeline (Centos 7) and doesn't proceed to build for the rest of the platforms.

Reply:
ok, but manually merge the patch without adding this change

Reply:
Cool, I'll revert only this newline change and merge.